index.md

```diff
@@ -26,6 +28,45 @@ like circuit breakers or request rates.
 Kiali also includes Jaeger Tracing
 to provide distributed tracing out of the box.
 
+## Getting started on OpenShift
+
+Kiali is still in development. Snapshots releases are pushed on Dockerhub from the CI pipeline.
+
+To deploy Kiali to your Istio-enabled OpenShift cluster you can run the following. Kiali currently requires Istio version 0.7.1.
+
+```
+export NAMESPACE=istio-system
+ 
+oc create -f https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml -n ${NAMESPACE}
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | \
+   IMAGE_NAME=kiali/kiali \
+   IMAGE_VERSION=latest \
+   NAMESPACE=${NAMESPACE} \
```

Comment:
I would just hardcode istio-system here - don't make it seem like its optional and can be any value. AFAIK, we have to be in istio-system with the rest of istio